src/features/pastelAddressBlock/PastelAddressBlock.tsx

```diff
@@ -0,0 +1,292 @@
+import dateformat from 'dateformat'
```

Comment:
If we should use daysjs, we wouldn't need this (can be replaced with https://day.js.org/docs/en/display/format)

Reply:
I have used dayjs now